pkg/service/roomallocator.go

```diff
@@ -51,8 +51,8 @@ func NewRoomAllocator(conf *config.Config, router routing.Router, rs ObjectStore
 	}, nil
 }
 
-func (r *StandardRoomAllocator) CreateRoomEnabled() bool {
-	return r.config.Room.CreateRoomEnabled
+func (r *StandardRoomAllocator) AutoCreateEnabled(context.Context) bool {
+	return r.config.Room.AutoCreate
```

Comment:
I guess we had been using the wrong field?

Reply:
no, i deprecated the old signal based room creation in the same diff because i didn't want to implement the new feature 2 ways